app/src/main/java/eu/kanade/tachiyomi/data/track/BaseTracker.kt

```diff
@@ -53,6 +55,14 @@ abstract class BaseTracker(
         get() = getUsername().isNotEmpty() &&
             getPassword().isNotEmpty()
 
+    override val isLoggedInFlow: Flow<Boolean>
+        get() = combine(
```

Comment:
Instead of using a `get()`, just save it to the val instead. This is a cold flow so it wont matter much. Plus saving it might be for better compose since it wont accidentally re-collect on run.